### ISSUE TITLE
Cannot paste into gimp

### ISSUE THREAD

**Issue by Mibou** [flameshot-org/flameshot#1544]
**Flameshot version**
v0.9.0

**Describe the bug**

The screenshot is saved to clip board and can be pasted into Chrome and Paint, but not into Gimp.
I need to paste to Mspaint and recopy from Mspaint, to be able to paste in Gimp.

**To Reproduce**

Take a screenshot to clipboard, open Gimp, paste.

![image](https://user-images.githubusercontent.com/1668996/114037309-29895000-9881-11eb-8439-09b43c5e6424.png)

**Expected behavior**

Should paste the image as it work from any other clipboard copyied image from any other application.

**System Information**

Windows 10

**Comment by borgmanJeremy** [comment]
Do you have jpeg for clipboard enabled? If so this is not a flameshot issue, many programs do not want to accept a jpeg being pasted in.

**Comment by Mibou** [comment]
I tryed to force useJpgForClipboard manually in flameshot.ini as I had this other issue ( https://github.com/flameshot-org/flameshot/issues/1543 ) and it didn't make it.

**Comment by mmahmoudian** [comment]
I could not reproduce this on Linux (Manjaro). I could paste the screenshot both with and without the "Use JPG format for clipboard (PNG default)"
GIMP 2.10.22
Flameshot 0.9.0

**Comment by sonulohani** [comment]
Is this related to this issue: https://gitlab.gnome.org/GNOME/gimp/-/issues/789

**Comment by Mibou** [comment]
Indeed, for anyone ending up on this bug, this is a bug from Gimp. Gimp 2.99 fixes it.
